.github/workflows/commit_checker.yaml

```
@@ -0,0 +1,28 @@
+name: 'Commit Message Check'
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - reopened
+      - synchronize
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - reopened
+      - synchronize
+  push:
+    branches:
+      - main
+
+jobs:
+  check-commit-message:
+    name: Check Commit Message
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check Commit Type
+        uses: gsactions/commit-message-checker@v1
+        with:
+          pattern: '^(Refs|Fixes) #YUPTOO-[0-9]+ - .+$'
+          error: 'Commit message should start with "Fixes #YUPTOO-<issue number> - " and end with a period(dot)'
```

```suggestion
          error: 'Commit message should start with "Fixes #YUPTOO-<issue number> - "'
```
Note that regex pattern won't force dot at the end. I never seen that it is mandatory to have dot at the end. Do you think we need to keep it?

---

Ohk, I thought regex is forcing dot. I will make the correction.